about.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+
+    <title>Document</title>
+</head>
+<body>
+    
+    <header>
+        <div class="top-head">
+        <ul class="top-desk">
+        <li><img class="fb" src="asset/facebook.png" alt="facebook"></li>
+        <li><img class="twit" src="asset/twitter.png" alt="twitter"></li>
+        <li>English</li>
+        <li>My page</li>
+        <li>Log out</li>
+        </ul>
+    </div>
+    <div class="desk-menu">
+        <img src="asset/cc global logo.png">
+        <ul class="desk-list">
```

Comment:
- [ ] please make this about main title bigger and color of \#EC5242.. remember we are trying to reproduce the original design
- [ ] kindly increase the text size below it so it is visible
![Screenshot (1295)_LI](https://github.com/Surafels/Capstone_Project/assets/65298048/14733481-03ea-4f87-a662-0aa8d50599c2)